paket.dependencies

```diff
@@ -7,6 +7,9 @@ nuget FSharp.Core content: none
 nuget FSharp.Compiler.Service == 41.0.3
 nuget FsUnit
 nuget FsCheck
+nuget System.IO.Abstractions 16.1.10
```

Comment:
`System.IO.Abstractions` is defined twice.
Any reason why you are locking the version?